### DISCUSSION TITLE
Windows Build Issues

### DISCUSSION THREAD

**Discussion by skr32** [flxzt/rnote#299] [General]
Hello,

I'm trying to build the Windows binary but I run into the following Error when I run meson setup:

meson.build:1:0: ERROR: Rust compiler rustc -C linker=cc can not compile programs.
error: linking with `cc` failed: exit code: 

[Here the full log: https://pastebin.com/ZK843Ehi](https://pastebin.com/TnbQgSpX)

I noticed the following line that it's looking for a file in the AppData folder:
= note: "cc" "/NOLOGO" "C:\\Users\\<user>\\AppData\\Local\\Temp\\rustcBrKBYo\\symbols.o"

**Comment by flxzt** [comment]
Hi.
The logs indicate that you are not using the right toolchain for compliation with mingw64. Try following these [steps](https://github.com/flxzt/rnote/blob/main/misc/building/rnote-windows-build.md) and change the rustc toolchain to `stable-gnu`.
